cumulusci/tasks/salesforce/tests/test_UpdateDependencies.py

```diff
@@ -197,6 +197,41 @@ def test_run_task__no_dependencies(self):
         task()
         api.assert_not_called()
 
+    @mock.patch(
+        "cumulusci.salesforce_api.metadata.ApiRetrieveInstalledPackages.__call__"
+    )
+    def test_run_task__duplicate_dependencies(self, ApiRetrieveInstalledPackages):
+        project_config = create_project_config()
+        project_config.config["project"]["dependencies"] = (
+            {"namespace": "package", "version": "1.0"},
+        )
+        project_config.get_github_api = mock.Mock()
+        project_config.get_static_dependencies = mock.Mock()
+        project_config.get_static_dependencies.return_value = [
+            {
+                "namespace": "dep1",
+                "version": "1.1",
+                "dependencies": [{"namespace": "dep2", "version": "1.1"}],
+            },
+            {"namespace": "dep2", "version": "1.1"},
+        ]
```

Comment:
This is okay, but since these are all namespace+version dependencies I think you could have just passed them in as the task's `dependencies` option without needing a mock.